packages/grid/_modules_/grid/hooks/root/useEvents.ts

```diff
@@ -87,34 +88,34 @@ export function useEvents(gridRootRef: React.RefObject<HTMLDivElement>, apiRef:
   useGridApiEventHandler(apiRef, GRID_COL_RESIZE_START, handleResizeStart);
   useGridApiEventHandler(apiRef, GRID_COL_RESIZE_STOP, handleResizeStop);
 
-  useGridApiEventHandler(apiRef, GRID_COLUMN_HEADER_CLICK, options.onColumnHeaderClick);
-  useGridApiEventHandler(
+  useGridApiOptionHandler(apiRef, GRID_COLUMN_HEADER_CLICK, options.onColumnHeaderClick);
+  useGridApiOptionHandler(
     apiRef,
     GRID_COLUMN_HEADER_DOUBLE_CLICK,
     options.onColumnHeaderDoubleClick,
   );
-  useGridApiEventHandler(apiRef, GRID_COLUMN_HEADER_OVER, options.onColumnHeaderOver);
-  useGridApiEventHandler(apiRef, GRID_COLUMN_HEADER_OUT, options.onColumnHeaderOut);
-  useGridApiEventHandler(apiRef, GRID_COLUMN_HEADER_ENTER, options.onColumnHeaderEnter);
-  useGridApiEventHandler(apiRef, GRID_COLUMN_HEADER_LEAVE, options.onColumnHeaderLeave);
-  useGridApiEventHandler(apiRef, GRID_COLUMN_ORDER_CHANGE, options.onColumnOrderChange);
+  useGridApiOptionHandler(apiRef, GRID_COLUMN_HEADER_OVER, options.onColumnHeaderOver);
```

Comment:
Not sure I understand these changes
https://github.com/mui-org/material-ui-x/blob/master/packages/grid/_modules_/grid/hooks/root/useEvents.ts#L97

Reply:
Ignore it. I didn't see that `useGridApiOptionHandler` was created in https://github.com/mui-org/material-ui-x/pull/1304. Rebasing here.